PrincessBrideTrivia/PrincessBrideTrivia/Program.cs

```diff
@@ -79,13 +85,15 @@ public static Question[] LoadQuestions(string filePath)
 
             string correctAnswerIndex = lines[lineIndex + 4];
 
-            Question question = new();
+            Question question = new Question();  //Added Question(); to intialize
```

Comment:
```suggestion
            Question question = new()
```
While this is not incorrect, the general rule of thumb is to use the simplified new();. But that can (as with practically every coding guideline) be argued.

https://learn.microsoft.com/dotnet/fundamentals/code-analysis/style-rules/ide0090?WT.mc_id=8B97120A00B57354